content/docs/reference/cmctl.md

```diff
@@ -34,17 +34,17 @@ This will also install shell completion.
 
 ### Manual Installation
 
-You need the `cmctl.tar.gz` file for the platform you're using, these can be
+You need the `cmctl` file for the platform you're using, these can be
 found on our
-[GitHub releases page](https://github.com/cert-manager/cert-manager/releases).
+[cmctl GitHub releases page](https://github.com/cert-manager/cmctl/releases).
 In order to use `cmctl` you need its binary to be accessible under
 the name `cmctl` in your `$PATH`.
 Run the following commands to set up the CLI. Replace OS and ARCH with your
 systems equivalents:
 
 ```console
-OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -fsSL -o cmctl.tar.gz https://github.com/cert-manager/cert-manager/releases/latest/download/cmctl-$OS-$ARCH.tar.gz
-tar xzf cmctl.tar.gz
+OS=$(go env GOOS); ARCH=$(go env GOARCH); curl -fsSL -o cmctl https://github.com/cert-manager/cmctl/releases/latest/download/cmctl_${OS}_${ARCH}
+chmod +x cmctl
 sudo mv cmctl /usr/local/bin
 ```
```

Comment:
I noticed I can now install `cmctl` using `go install`. It think it's an option that lots of people would be happy knowing about:

~~~suggestion
```

Alternatively, you can install `cmctl` using `go`:

```bash
go install github.com/cert-manager/cmctl/v2@latest
```
~~~